app/models/nextPayments/viewmodels/WYOClaimToAdjustViewModel.scala

```diff
@@ -29,8 +30,16 @@ case class WYOClaimToAdjustViewModel(adjustPaymentsOnAccountFSEnabled: Boolean,
     }
   }
 
-  def ctaLink = {
-    claimToAdjustTaxYear
+}
+
+object WYOClaimToAdjustViewModel {
+
+  def ctaLink(isAgent: Boolean): String = {
+    if (isAgent) {
+      "/report-quarterly/income-and-expenses/view/agents/adjust-poa/start"
```

Comment:
could fix: why not get this url from routes something like this  - `controllers.routes.CreditsSummaryController.showAgentCreditsSummary(calendarYear).url`?

Reply:
The page it's redirecting to is not created yet and I didn't want to make dummy controllers/routes to keep the compiler happy but your suggestion would be an improvement for future or if the other stuff is merged

Reply:
cool 😎